src/App.tsx

```diff
@@ -15,48 +17,133 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+type Good = string;
+type Sort = SortType | '';
```

Comment:
these types are redundant. You can use string instead of `Good` and it will be more clear to read. And you can add NONE sort type with `''` value to `SortType` enum